pkg/daemon/ceph/osd/volume_test.go

```diff
@@ -354,6 +620,46 @@ func TestInitializeBlockPVC(t *testing.T) {
 	_, _, _, err = a.initializeBlockPVC(context, devices, false)
 	assert.NotNil(t, err)
 
+	// Test for OSD on LV-backed PVC where Ceph does not support raw mode.
+	// Expect no commands to be used.
+	executor.MockExecuteCommandWithCombinedOutput = func(command string, args ...string) (string, error) {
+		logger.Infof("%s %v", command, args)
+		return "", errors.Errorf("unknown command %s %s", command, args)
+	}
+	// Test with CephVersion{Major: 14, Minor: 2, Extra: 7}
+	clusterInfo = &cephclient.ClusterInfo{
+		CephVersion: cephver.CephVersion{Major: 14, Minor: 2, Extra: 7},
+	}
+	a = &OsdAgent{clusterInfo: clusterInfo, nodeName: "node1"}
+	devices = &DeviceOsdMapping{
+		Entries: map[string]*DeviceOsdIDEntry{
+			"data": {Data: -1, Metadata: nil, Config: DesiredDevice{Name: "/mnt/set1-data-0-rpf2k"}},
+		},
+	}
+	_, _, _, err = a.initializeBlockPVC(context, devices, true)
+	assert.NotNil(t, err)
+	logger.Infof("error message %v", err)
+
+	// Test for OSD on LV-backed PVC where Ceph does not support raw mode.
```

Comment:
Isn't this the same as the previous test? 👀

Reply:
Sorry. It is the same test and I delete a duplicate test.